src/main/java/org/myrobotlab/service/InMoovArm.java

```diff
@@ -261,22 +246,6 @@ public void waitTargetPos() {
   // FIXME - releasePeers()
   public void release() {
     disable();

```

Comment:
disable is different than release is it not?  seems like this really should just be removed and we should use releaseService here?

Reply:
whoa .. not that i see it I don't know how the hell this came about..
it should be 
public void releaseService {
     super.releaseService();
     disable();

before .. I wrote a long comment about the problems of using polymorphism for framework lifecycle.   Its problematic for the developer to always remember to call super.releaseService() ..
So I said "blah blah mebbe the framework should always call some other method name which effectively does super.releaseService() and **THEN**  call this function.  That way developers don't need to remember to call the super .. but I then realized this is just whack ...

Ahahah  5 years ago - there was never a releaseService and this NEVER got called .... ever .. 

fixing after 5 years ..

Reply:
pushed